lib/generators/rolemodel/ci/github/templates/rspec.yml

```diff
@@ -0,0 +1,35 @@
+name: "RSpec"
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["main"]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:15-alpine
+        ports:
+          - "5432:5432"
+        env:
+          POSTGRES_DB: rails_test
+          POSTGRES_USER: rails
+          POSTGRES_PASSWORD: password
+    env:
+      RAILS_ENV: test
+      DATABASE_URL: "postgres://rails:password@localhost:5432/rails_test"
```

Comment:
We had issues using `DATABASE_URL` with parallel testing. 
Could you change to https://github.com/RoleModel/gimme-credit/blob/main/.github/workflows/ci.yml#L42-L53

It does require some updates to the database.yml for some reason though https://github.com/RoleModel/gimme-credit/blob/main/config/database.yml#L60-L65